lumigator/infra/mzai/helm/lumigator/templates/gen_secrets.yaml

```diff
@@ -0,0 +1,13 @@
+{{- include "lumigator.consts" . -}}
+{{- if .Values.mistralKey -}}
+{{- if .Values.mistralSecretName -}}
+{{- fail "A Mistral key and a Secret name cannot be provided at the same time" -}}
+{{- end -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "lumigator.fullname" . }}
```

Comment:
the secret name should be `{{ include "lumigator.fullname" . }}-mistral`

Reply:
Also please include:

```
  labels:
    {{- include "lumigator.labels" . | nindent 4 }}
```

Reply:
Ok to both